media-libs/alembic/alembic-1.7.7.recipe

```diff
@@ -6,15 +6,29 @@ of scenes into baked geometry is exactly analogous to the distillation of \
 lighting and rendering scenes into rendered image data."
 HOMEPAGE="http://www.alembic.io/"
 COPYRIGHT="2009-2016 Sony Pictures Imageworks
-	2009-2016 Industrial Light and Magic"
+	2009-2016 Industrial Light and Magic
+	2010-2012 Christopher Horvath
+	2011-2012 Luxology LLC
+	2011-2013 Autodesk, Inc.
+	2011-2018 Side Effects Software Inc.
+	2011-2013 Mark Elendt, Senior Mathematician, Side Effects Software Inc.
+	2011-2013 Nicholas Yue
+	2011 Fabric Engine, Inc.
+	2011 Luma Pictures
+	2012 Solid Angle S.L.
+	2012 Zhicheng YE
+	2013 Digital Domain 3.0, Inc.
+	2016-2018 Tim Yanalunas
+	2016-2018 Blizzard Entertainment, Inc.
+	2018 J Cube, Inc."
 LICENSE="ALEMBIC"
-REVISION="3"
+REVISION="1"
 SOURCE_URI="https://github.com/alembic/alembic/archive/$portVersion.tar.gz"
-CHECKSUM_SHA256='ab0c727bfc4aedfe81c2365e604fb457056dc66909575ccfe19ed406e67c002e'
+CHECKSUM_SHA256='831b016afa9e66187b66819a02c0bd198b0f30e501a3552d0634796728cf1bff'
 PATCHES="alembic-$portVersion.patchset"
 
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
-SECONDARY_ARCHITECTURES="?x86"
+SECONDARY_ARCHITECTURES="x86"
 
 PROVIDES="
```

Comment:
```diff
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+        commandBinDir=$prefix/bin
+fi
+
PROVIDES="
```
(This should do the trick, with the [`PATCH()`](https://github.com/haikuports/haikuports/pull/2405#issuecomment-377707251) function and the alternate definition of [`ILMBASE_INC_DIR`](https://github.com/haikuports/haikuports/pull/2405/files#r178426291).)